src/runtime_src/core/pcie/tools/xbmgmt/xbmgmt.cpp

```diff
@@ -113,13 +113,23 @@ static inline bool isHiddenSubcmd(const std::string& cmd)
 
 static void printHelp(void)
 {
+    std::vector<std::string> expert_subCmd =
+        { "reset", "clock", "partition", "config", "nifd" };
```

Comment:
it'd be better to do it the other way around - have a list of normal cmds and anything not in the list would be for expert. i'd think that normal cmds are committed / stable ones, which can be easily changed, while expert cmd comes and goes... so that we don't need to change this list often.